freelist_hmap.go

```diff
@@ -108,6 +110,33 @@ func (f *freelist) hashmapGetFreePageIDs() []common.Pgid {
 
 // hashmapMergeSpans try to merge list of pages(represented by pgids) with existing spans
 func (f *freelist) hashmapMergeSpans(ids common.Pgids) {
+	common.Verify(func() {
+		ids1 := f.idsFromFreemaps()
+		ids2 := f.idsFromForwardMap()
+		ids3 := f.idsFromBackwardMap()
```

Comment:
could you give those variables a proper name please? it's a bit hard to follow the logic below without the backward/forward in mind

Reply:
Updated. I was thinking the short names `ids1/ids2/ids3` are better.